neo/test/iotest/test_intanio.py

```diff
@@ -19,6 +19,8 @@ class TestIntanIO(
         "intan/intan_rhd_test_1.rhd",
         "intan/intan_fpc_test_231117_052630/info.rhd",
         "intan/intan_fps_test_231117_052500/info.rhd",
+        "intan/intan_fpc_rhs_test_240329_091536/info.rhs",
+        "intan/intan_fps_rhs_test_240329_091637/info.rhs",
```

Comment:
Did you add new files to gin? I am planning to give another review to this.

Reply:
No I typed them backwards, but we are missing the fps files from gin. Your PR only ad the fpc files. So I was trying to set-up gin again so that you'd wouldn't have to add the last folder set.

Reply:
Let's do this. I also wanted to ask you, could you generrate a small dataset for the option when the "classical" format is partitioned in time?

Reply:
Sure. I'll slack it to you in a couple minutes. Is three minutes enough? (ie 3 files).